documentation/modules/ROOT/pages/modules/cdc-mysql-connector/c_mysql-connector-events.adoc

```diff
@@ -314,32 +316,53 @@ CREATE TABLE customers (
 }
 ----
 
+The schema portion of this event’s _value_ shows the schema for the envelope, the schema for the source structure (which is specific to the MySQL connector and reused across all events), and the table-specific schemas for the `before` and `after` fields.
+
+[TIP]
+====
+The names of the schemas for the `before` and `after` fields are of the form `<logicalName>.<tableName>.Value`, and thus are entirely independent from all other schemas for all other tables.
+ifndef::cdc-product[]
+This means that when using the {link-prefix}:{link-avro-serialization}[Avro Converter], the resulting Avro schemas for each table in each logical source have their own evolution and history.
+endif::cdc-product[]
+====
+
+The `payload` portion of this event’s _value_ shows the information in the event, namely that it is describing that the row was created (because `op=c`), and that the `after` field value contains the values of the new inserted row's `id`, `first_name`, `last_name`, and `email` columns.
+
+[TIP]
+====
+It may appear that the JSON representations of the events are much larger than the rows they describe. This is because the JSON representation must include the schema and the payload portions of the message.
+ifndef::cdc-product[]
+However, by using the {link-prefix}:{link-avro-serialization}[Avro Converter], you can dramatically decrease the size of the actual messages written to the Kafka topics.
```

Comment:
Without the note on Avro the tip reads a bit awkward, IMHO. As it describes a shortcoming, but no mitigation. Maybe skip it altogether downstream for the time being?